src/security/srv_acl.c

```diff
@@ -758,3 +758,12 @@ ds_sec_cont_can_read_data(uint64_t cont_capas)
 {
 	return (cont_capas & CONT_CAPA_READ_DATA) != 0;
 }
+
+uint64_t
+ds_sec_get_rebuild_cont_capabilities(void)
+{
+	/*
+	 * Internally generated rebuild container handles can read data
+	 */
+	return CONT_CAPA_READ_DATA;
```

Comment:
Is this only read or do we need write as well? Also does it get or set any props?

Reply:
I believe (based on some discussions) that rebuild only goes through code paths shared with the client for the object read. That would explain why rebuild tests didn't start failing until we started checking for read permissions here. Get and set prop permissions are checked pretty high up in the client entry point.

Reply:
Looks like we've passed Functional tests, which included the rebuild cases we were failing before.